src/components/RecipientSidePanel.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Box } from '@chakra-ui/react';
+
+export default function RecipientSidePanel() {
+  return (
+    <div style={{ paddingTop: '20px', paddingLeft: '40px' }}>
+      <Box
+        as="button"
```

Comment:
Why are so many of the `Box` components set as buttons? There should only be one button in this component